willa/tind/format_tind_context.py

```diff
@@ -0,0 +1,65 @@
+"""
+Format Tind fields for context in Chat response
+"""
+
+DISPLAY_MAPPINGS: dict = {
+    'tind_id': 'Tind ID:',
+    'isPartOf': 'Project Name:',
+    'title': 'Title:',
+    'contributor': 'Contributor:'
+}
+
+
+def get_tind_url(tind_id: str) -> str:
+    """Create a Tind link using the Tind ID
+
+    :param: tind_id
+    :returns: URL to Tind record
+    """
+
+    tind_url = f"https://digicoll.lib.berkeley.edu/record/{tind_id}"
+    return tind_url
+
+
+def get_tind_context(docs: list) -> str:
+    """Provide formatted Tind metadata for chatbot responses.
+
+    :param: Array of documents from chatbot 
+    :returns: Formatted text of select Tind data
+    """
+
+    tind_data = ''
+    tind_ids = {}
+    for doc in docs:
+        tind_id = doc.metadata['tind_metadata']['tind_id']
+        if tind_id in tind_ids:
+            continue
+
+        tind_data += f"\n\n{process_fields(doc.metadata['tind_metadata'])}\n___________\n\n"
+        tind_ids[tind_id] = tind_id
+
+    return tind_data
+
+
+def process_fields(tind_rec: dict) -> str:
```

Comment:
i think there a re a couple ways to refactor this module:

1. Using template strings: https://docs.python.org/3/library/string.html#template-strings
2. Using list comprehensions (see below)